auth/src/main/java/org/aerogear/mobile/auth/AuthService.java

```diff
@@ -37,6 +38,7 @@
     private ServiceConfiguration serviceConfiguration;
     private KeycloakConfiguration keycloakConfiguration;
     private AuthServiceConfiguration authServiceConfiguration;
+    private BrowserConfiguration browserConfiguration;
```

Comment:
where is this `browserConfiguration` being initialised?

Reply:
I forgot to remove this, my bad.  The `BrowserConfiguration` can be initialised by the developer and passed into the `AuthService`

Reply:
ok, but what happens if developer doesn't init one?